src/components/molecules/NavBar/NavBar.scss

```diff
@@ -3,7 +3,7 @@
 $border-radius: 28px;
 
 .navbar-container {
-  background: $black;
+  @include background-color($black);
```

Comment:
NOTE: function sets the needed CSS --variables for the descendant indicator border to preclude the direct parent's paint.